db/version_set.h

```diff
@@ -601,12 +601,13 @@ class Version {
   Version* next_;               // Next version in linked list
   Version* prev_;               // Previous version in linked list
   int refs_;                    // Number of live refs to this version
+  EnvOptions env_options_;
```

Comment:
can it be const? might be beneficial for readability so people can feel safe accessing it outside of lock